src/angular/14-building-order-form/child-component/menu-items.component-props.ts

```diff
@@ -8,14 +8,12 @@ interface Item {
 @Component({
   selector: 'pmo-menu-items',
   templateUrl: './menu-items.component.html',
-  styleUrls: ['./menu-items.component.less']
+  styleUrls: ['./menu-items.component.less'],
 })
 export class MenuItemsComponent implements OnInit {
-  @Input() items: Item[];
+  @Input() items?: Item[];
```

Comment:
I think I didn't go with this approach because it would require changing a lot of the exercises we already have, because of the types we have as example. The parent component was probably sending undefined/null down to this component. I could open an issue to fix all undefined/null types later and make the code more "strict".

Reply:
At least the code is aware that the value is possibly undefined and is treating it accordingly